### PR TITLE
ZK-4839: za11y: hover navitem without children (JS Error)

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4839Test.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+
+public class B96_ZK_4839Test extends WebDriverTestCase {
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4839Test.java:18]: WcagTestOnly

File: zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4839Test.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	@Test
+	public void test() {
+		connect();
+		mouseOver(jq(".z-icon-car"));
+		waitResponse();
+		Assert.assertFalse("You should not see any js error", hasError());
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4839Test.java:24]: assertNoJSError()

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**